src/prefect/server/api/root.py

```diff
@@ -11,3 +15,21 @@
 async def hello():
     """Say hello!"""
     return "👋"
+
+
+@router.get("/ready")
+async def perform_readiness_check(
+    db: PrefectDBInterface = Depends(provide_database_interface),
+):
+    is_db_connectable = await db.is_db_connectable()
+
+    if is_db_connectable:
+        return JSONResponse(
+            status_code=status.HTTP_200_OK,
+            content=jsonable_encoder({"status": "OK"}),
+        )
+
+    return JSONResponse(
+        status_code=status.HTTP_503_SERVICE_UNAVAILABLE,
+        content=jsonable_encoder({"status": "Database is not available"}),
```

Comment:
Do we really need the `jsonable_encoder` call? I'm pretty sure we do not use this elsewhere

Nit: Can we do "message" or "details" instead of "status"? Status feels like an enum to me.

Reply:
errrm maybe we don't need `jsonable_encoder`, i did pull this out from examples in this repo, but taking a look at the fastapi docs, `jsonable_encoder` is more for pydantic models that first need to be converted to a dict. in our case, the responses are hardcoded dicts so dont think we need em

>Nit: Can we do "message" or "details" instead of "status"? Status feels like an enum to me.
works for me